verticaltabs.js

```diff
@@ -723,20 +727,29 @@ VerticalTabs.prototype = {
 
   clearFind: function () {
     this.document.getElementById('find-input').value = '';
+    if (this.visibleTabs) {
+      for (let i = 0; i < this.visibleTabs.length; i++) {
+        let tab = this.visibleTabs[i];
+        if (tab.getAttribute('pinned') === 'true') {
```

Comment:
Maybe leave a comment about why we're doing this…

(We could also filter only the pinned tabs, and then set it on all of _them_, like:
```js
this.visibleTabs
    .filter(tab => tab.getAttribute('pinned') === 'true')
    .forEach(tab => {tab.setAttribute('hidden', false);});
```
if you felt like functional programming…  :wink:)